Sources/SafeDITool/SafeDITool.swift

```diff
@@ -52,6 +52,13 @@ struct SafeDITool: AsyncParsableCommand, Sendable {
 			throw ValidationError("Must provide 'swift-sources-file-path', '--include', or '--include-file-path'.")
 		}
 
+		async let existingGeneratedCode: String? = Task.detached {
+			if let dependencyTreeOutput {
+				try? String(contentsOf: dependencyTreeOutput.asFileURL, encoding: .utf8)
```

Comment:
Do we need to log if unable to initialize a string with the content of the tree output? Anyway, it should be `nil` for initialization errors, or the file doesn't exist initially.

Reply:
I don't think so – if we can't read it for whatever reason we'll just write a new one.